sdk/communication/Azure.Communication.Chat/src/autorest.md

```diff
@@ -11,7 +11,7 @@ If any of the new objects needs to be overwritten, add the required changes to t
 
 ``` yaml
 input-file:
-    -  communicationserviceschat.json
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/communication/data-plane/Microsoft.CommunicationServicesChat/stable/2021-03-07/communicationserviceschat.json
```

Comment:
instead of referencing the master branch, you should point to the specific commit

Reply:
Looks like the generation check is already failing for this https://dev.azure.com/azure-sdk/public/_build/results?buildId=808544&view=logs&j=b70e5e73-bbb6-5567-0939-8415943fadb9&t=72c41882-9d21-5580-4869-7f3ea1bd9c0a. @juancamilor can you please put in a fix for this to unblock the builds? I don't know why the PR validation didn't hit this but the CI build and the core builds are now hitting this error.

Reply:
@weshaggard  we just opened a new PR with that change, but the ci validation is now complaining with :
Build log #L319
Generated code is not up to date.
https://github.com/Azure/azure-sdk-for-net/pull/19879/checks?check_run_id=2206538211
we are looking at it right now.

Reply:
ok, the fix is in master now @weshaggard

Reply:
cc @pakrym 

Yes it is my understanding we require a sha instead of a branch to ensure the generation is more stable and doesn't break the generation as the branch updates.

Reply:
Correct, we want reproducible generation independently of the state of the master in the `azure-rest-api-specs` repo.